.github/workflows/deploy_awslambda.yml

```diff
@@ -21,8 +21,8 @@ jobs:
 
   get-external-artifacts:
     permissions:
-      actions: read  # for dawidd6/action-download-artifact to query and download artifacts
-      pull-requests: read  # for dawidd6/action-download-artifact to query commit hash
+      actions: read  # for download-artifact to query and download artifacts (unsure if needed since moving from dawidd6/actions-download-artifact?)
```

Comment:
Could we test this somewhere else so we can know for sure?

Reply:
Looking closer at that workflow it does have a test mode, so I can test it out.  I think it only needs `actions: read`.  I'm also going to try and eliminate the "download external and upload locally, then redownload" business.